src/features/projects/components/projectDetails/getProjectClassification.ts

```diff
@@ -1,21 +1,21 @@
 function GetProjectClassification(classification: any) {
   let parseClassification;
   switch (classification) {
-    case 'natural_regeneration':
+    case 'natural-regeneration':
       parseClassification = 'Natural Regeneration';
       break;
-    case 'managed_regeneration':
+    case 'managed-regeneration':
       parseClassification = 'Managed Regeneration';
       break;
-    case 'large_scale_planting':
+    case 'large-scale-planting':
       parseClassification = 'Large Scale Planting';
       break;
     case 'agroforestry':
       parseClassification = 'Agroforestry';
       break;
     default:
       parseClassification = '';
-  }
+  } 
   return parseClassification;
 }
 
```

Comment:
This needs to be translated. you don't really need a switch case just make `t(project.classification)` and add all possible values to translation file. you might want to check if `project.classification` exists first.